README.md

```diff
@@ -78,8 +78,8 @@ Group meeting times are listed below:
 
 - US:   Weekly on Wednesdays at 10:00am UTC-7 (see your timezone
   [here](https://time.is/1000_today_in_PT?CNCF_Security_TAG_US_Meeting))
-- EMEA: Bi-weekly on Wednesdays at 01:00pm London (see your timezone
-  [here](https://time.is/1300_today_in_London?CNCF_Security_TAG_EMEA_Meeting))
+- EMEA: Bi-weekly on Wednesdays at 12:00pm UTC+0/01:00pm UTC+1 (see your timezone
```

Comment:
Can we make it clear when it's UTC+0 vs UTC+1? Otherwise LGTM

Reply:
Yes. Added explanation describing its UTC+1 during daylight savings.